.gitignore

```diff
@@ -1,133 +0,0 @@
-#experiments/*
```

Comment:
why did you delete this file?

Reply:
I thought I just removed it from my commit, will amend